src/PropertySuggester/GetSuggestions.php

```diff
@@ -88,12 +89,17 @@ public function execute() {
 		);
 
 		if ( $params->entity !== null ) {
-			$suggestions = $suggestionGenerator->generateSuggestionsByItem(
-				$params->entity,
-				$params->suggesterLimit,
-				$params->minProbability,
-				$params->context
-			);
+			try {
+				$suggestions = $suggestionGenerator->generateSuggestionsByItem(
+					$params->entity,
+					$params->suggesterLimit,
+					$params->minProbability,
+					$params->context
+				);
+			} catch ( InvalidArgumentException $ex ) {
+				$id = $params->entity;
+				$this->dieWithError( "Invalid id: $id", 'no-such-entity', [ 'id' => $id ] );
```

Comment:
That's way beyond scope of this PR. I want this done to reduce logspam not to change PropertySuggestor behavior.